helm/circles-graphprotocol/templates/ethereum-networks.yaml

```diff
@@ -6,4 +6,8 @@ metadata:
 
 data:
   networks: >-
-    {{ .Values.network }}:http://{{ .Values.rpcNode }}:{{ .Values.rpcPort}}
+   {{ if eq .Values.env "stg" }}
+      {{ .Values.network }}:https://{{ .Values.rpcNode }}
```

Comment:
no port needed for chiado?

Reply:
nop, because we are using the public endpoint directly